surface/include/pcl/surface/mls.h

```diff
@@ -49,32 +49,163 @@
 #include <pcl/surface/eigen.h>
 #include <pcl/surface/processing.h>
 #include <map>
+#include <boost/function.hpp>
 
 namespace pcl
 {
+
+  const double MLS_PROJECTION_CONVERGENCE_TOLERANCE = 1e-8;
+  const double MLS_MINIMUM_PRINCIPLE_CURVATURE = 1e-5;
```

Comment:
I'm not sure about having this here...

Reply:
Where would you like them? I could also do away with them and put the values in there respected places.

Reply:
I'd say so. They both are implementation details and each is used only in a single place. Just move these const definitions into their respective functions, nobody will miss them in the main header.